INSTALL.rst

```diff
@@ -85,11 +85,10 @@ exe/wheel installers hosted on
 If you want to compile psutil from sources you'll need **Visual Studio**
 (Mingw32 is no longer supported):
 
-* Python 2.6, 2.7: `VS-2008 <http://www.microsoft.com/en-us/download/details.aspx?id=44266>`__
-* Python 3.3, 3.4: `VS-2010 <http://www.visualstudio.com/downloads/download-visual-studio-vs#d-2010-express>`__
```

Comment:
why remove instructions for 2.7 and 3.4?

Reply:
Oops, mistaken removal of 2.7 along with 2.6, and 3.4 is still there, just 3.3 removed.